### ISSUE TITLE
Update for L40S vGPU Support

### ISSUE THREAD

**Issue by jxdn** [NVIDIA/gpu-operator#719]
Hi , 

Can you add the L40S vGPU config..?

https://github.com/NVIDIA/gpu-operator/commit/214083b22a1a8fd3b4bf59a0e7d7f82408af6837

```[tasklist]
### Tasks
- [ ] https://github.com/NVIDIA/gpu-operator/pull/720
- [ ] https://github.com/NVIDIA/gpu-operator/pull/720
```

**Comment by jxdn** [comment]
https://github.com/NVIDIA/gpu-operator/pull/720

**Comment by cdesiniotis** [comment]
https://github.com/NVIDIA/gpu-operator/pull/799 has been merged into main. This commit added entries for L40S. This will be included in our upcoming 24.6.0 release.

I am closing this issue as fixed.
